src/lex.ll

```diff
@@ -706,64 +714,186 @@ lCComment(SourcePos *pos) {
     Error(*pos, "unterminated comment");
 }
 
+static void lNextValidChar(SourcePos *pos, char const*&currChar) {
+    while (*currChar == ' ') {
+        ++pos->last_column;
+        currChar++;
+    }
+}
+
+/** Handle pragma directive to unroll loops.
+*/
+static void lPragmaUnroll(YYSTYPE *yylval, SourcePos *pos, std::string fromUserReq, bool isNounroll) {
+
+    const char *currChar = fromUserReq.data();
+    yylval->pragmaAttributes = new PragmaAttributes();
+    yylval->pragmaAttributes->aType = PragmaAttributes::AttributeType::pragmaloop;
+    int count = -1;
+
+    lNextValidChar(pos, currChar);
+
+    if (isNounroll) {
+        if (*currChar == '\n') {
```

Comment:
The line may end by ``\r\n``. Would that be a problem?

Reply:
Since we treat \r as whitespace, \r\n won.\'t be  a problem. In theory we should ignore the \r as whitespace in 'lNextValidChar' and get to \n